content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -59,25 +56,34 @@ If a :abbr:`BoM (Bill of Materials)` does not already exist for the product, sel
 :guilabel:`New` to configure a new :abbr:`BoM (Bill of Materials)`.
 
 .. image:: reordering_rules/bom-smart-button.png
-   :align: center
    :alt: The Bill of Materials smart button on a product form.
 
 Create new reordering rules
 ===========================
 
-To create a new reordering rule, navigate to :menuselection:`Inventory app --> Configuration -->
-Reordering Rules`, then click :guilabel:`New`, and fill out the new line as follows:
+To create a new reordering rule, navigate to
+:menuselection:`Inventory app --> Operations --> Replenishment`, then click :guilabel:`New`,
+and fill out the following fields for the new reordering rule line item:
 
 - :guilabel:`Product`: The product that is replenished by the rule.
-- :guilabel:`Location`: The location where the product is stored.
+- :guilabel:`On Hand`: The number of units currently available in inventory.
+- :guilabel:`Forecast`: The number of units expected to be available in inventory after all orders
+  are taken into account.
+- :guilabel:`Route`: The
+  :doc:`manufacturing route <../../shipping_receiving/daily_operations/use_routes>` for this product
 - :guilabel:`Min Quantity`: The minimum quantity that can be forecasted without the rule being
   triggered. When forecasted stock falls below this number, a replenishment order for the product is
   created.
-- :guilabel:`Max Quantity`: The maximum quantity that stock is replenished up to.
-- :guilabel:`Multiple Quantity`: Specify if the product should be replenished in batches of a
-  certain quantity (e.g., a product could be replenished in batches of 20).
-- :guilabel:`UoM`: The unit of measure used for reordering the product. This value can simply be
-  `Units` or a specific unit of measurement for weight, length, etc.
+- :guilabel:`Max Quantity`: The maximum quantity at which the stock is replenished.
+- :guilabel:`To Order`: The number of units, according to the *UoM* that should be replenished for
+  this reordering rule.
+- :guilabel:`UoM`: The *unit of measurement* used for reordering the product. This value can simply
+  be `Units` or a more specific unit of measurement for weight, length, etc.
+
+Additional fields can be accessed by clicking the
```

Comment:
Good suggestion.
Can throw that in a **Tip** admonition if you like to add a splash of color to the page 😉